.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - "0.10"
+  - "4"
```

Comment:
@paulfalgout For confirmation...you want me to replace the first line with the second? Or do you want me to just add the second line?